### ISSUE TITLE
[Feature Request] Add sort by release date to the game list

### ISSUE THREAD

**Issue by Thevenin-Jonathan** [VRPirates/rookie#113]
Hi !

Thanks for this beautiful soft ! Do you think is possible to add possibly to sort byrelease date into game list ? It would be a great improvment !

Thank you again ;)

**Comment by Fenopy** [comment]
Is this different than the current sort by release date rookie does?

**Comment by Thevenin-Jonathan** [comment]
> Est-ce différent de l’actuel tri par date de sortie des recrues ?

Hum, currently, I can only sort by date of last update. I don't have a sorting function by release date of the game.

**Comment by floppa9** [comment]
Yeah that could be awesome!!! pls add this

**Comment by Thevenin-Jonathan** [comment]
> Yeah that could be awesome!!! pls add this

Yes! We can't differentiate between a new game being added and an update to a game. It would be cool to add a visual element that shows us the new game additions.

**Comment by Doxorn** [comment]
I am missing this feature as well. I would like to see the newly released games that were added to the list (not the most recently updates ones).

**Comment by Soliver84** [comment]
Yes I am missing this feature as well
